sites/projectSelection.php

```diff
@@ -24,13 +24,19 @@
     <div align="center" style="width: 100%;">
         <div id="projekte" style="width: 90%;">
             <?php
-                $testArray = array("p1","p2","p3");
-                foreach($testArray as $project){
-                    echo "<a href=\"./votingPage.php?project=$project\">";
-                    echo "<div class=\"card projectButton\">";
-                    echo"<p>$project</p>";
-                    echo"</div>";
-                    echo"</a>";
+                $path = "../uploads";
```

Comment:
Wie gesagt ja später durhc Datenbank

Reply:
Ja steht ja schon in den TODOs müssen wir noch abarbeiten